src/epics/earthEngine.js

```diff
@@ -148,14 +147,12 @@ export const loadCollection = action$ =>
             );
 
             if (token && token.status === 'ERROR') {
-                return setAlert(
-                    createAlert(
-                        i18n.t(token.message),
-                        i18n.t(
-                            'To show this layer you must first sign up for the Earth Engine service at Google. Please check the DHIS 2 documentation.'
-                        )
-                    )
-                );
+                return setAlert({
+                    warning: true,
+                    message: i18n.t(
+                        'To show this layer you must first sign up for the Earth Engine service at Google. Please check the DHIS 2 documentation.'
```

Comment:
Suggested content: "This layer requires a Google Earth Engine account. Check the DHIS2 documentation for more information".

Reply:
Fixed.